Dockerfile

```diff
@@ -12,8 +12,8 @@ RUN python -m pip install --upgrade pip \
 RUN mkdir -p /code
 WORKDIR /code
 
-ADD . /code/
+COPY . /code/
 
 RUN pip install /code
 
-CMD ["python --version"]
+CMD ["/usr/bin/python", "--version"]
```

Comment:
Why the full path?

Reply:
python --version in one call wasn't working and I tested like that... but you are right just python should work